packages/dd-trace/src/config.js

```diff
@@ -520,6 +520,7 @@ class Config {
     this._setValue(defaults, 'lookup', undefined)
     this._setValue(defaults, 'inferredProxyServicesEnabled', false)
     this._setValue(defaults, 'memcachedCommandEnabled', false)
+    this._setValue(defaults, 'middleware', true)
```

Comment:
This name seems too simple and easy to confuse the purpose. How about `enableMiddlewareTracing`?

Reply:
I prefer this name, yes.